plugins/generated.nix

```diff
@@ -0,0 +1,44 @@
+{ pkgs, lib, config, ... }:
+
+with lib;
+
+let
+
+  helpers = import ./helpers.nix { inherit lib config; };
+
+  # names inserted here must match the name of the package in pkgs.vimExtraPlugins
+  plugs = with pkgs.vimExtraPlugins; [
+    "nvim-comment-frame"
+    "vim-printer"
+    "vim-easy-align"
+    "plantuml-syntax"
+    "gruvbox"
+    "nest-nvim"
+    "plenary-nvim"
+    "nvim-ts-context-commentstring"
+    "telescope-nvim"
+    "indent-blankline-nvim"
+    { name = "LuaSnip"; setup = false; }
+    { name = "lsp-signature-nvim"; setup = false; }
+  ];
+
+in with helpers; {
+  imports = lib.forEach plugs
+    (p:
+    let
+      name = if isString p then p else p.name;
+      setup =
+        if isString p then 
+          ""
+        else 
```

Comment:
Maybe add

```nix
if isString p.setup then p.setup
```

so that one could specify the setup function manually

Reply:
Sure